targets/java/source_shared/src/main/java/com/playfab/internal/PlayFabHTTP.java

```diff
@@ -101,14 +101,15 @@ public static String receive(InputStream in) throws IOException {
         return recieved.toString();
     }
 
-    public static PlayFabError GeneratePfError(int httpCode, PlayFabErrorCode pfErrorCode, String errorMessage, Map<String, List<String>> errorDetails) {
+    public static PlayFabError GeneratePfError(int httpCode, PlayFabErrorCode pfErrorCode, String errorMessage, Map<String, List<String>> errorDetails, Integer retryAfterSeconds) {
```

Comment:
Will this create an implicit cast between uint? and Integer?

Reply:
For Java I left the fields as an Integer, as that is how that lang handles nullable numbers